src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -8,7 +8,7 @@ Locale: en-US
 
 # AzureRM.Sql Module
 ## Description
-This topic displays help topics for the [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/) Cmdlets.
```

Comment:
@akromm I was under the impression that this was the source. Do you remember the command you used to regenerate the markdown help?